### PR TITLE
Closes #647 Announce deprecation of main and latest CDN buckets

### LINKED ISSUE #647: Add CDN information to main ahead of last 2.x release from main branch.
## Conditions of satisfaction
- [x] Readme on main branch says that the main bucket in the cdn is deprecated.

### PR DESCRIPTION
**Pull request by trackleft**
Closes #647 

This should be included in the last release to update the "main" bucket `https://cdn.digital.arizona.edu/lib/arizona-bootstrap/main` in our CDN.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,4 +25,66 @@ Arizona Bootstrap is built off of Bootstrap 4, and UA Bootstrap is built off of
...
+```
+
+#### The `<version bucket>` parameter can be any of the following
+- `main` Deprecated
+- `2.x` Development version of Arizona Bootstrap 2
+- `5.x` Development version of Arizona Bootstrap 5
+- A release version number, Example: `2.0.19` See [the release
+page](https://github.com/az-digital/arizona-bootstrap/releases) for a full list
+of possible options. Remove the `v` from the version number.
+- `latest-2.x` The latest stable version of Arizona Bootstrap 2
```
Comment (trackleft) [inline README.md:54]:
```suggestion
- `latest-2.x` The latest tagged version of Arizona Bootstrap 2
```

File: README.md [hunk 2]
```diff
@@ -25,4 +25,66 @@ Arizona Bootstrap is built off of Bootstrap 4, and UA Bootstrap is built off of
...
+- `latest-5.x` The latest stable version of Arizona Bootstrap 5
```
Comment (trackleft) [inline README.md:55]:
```suggestion
- `latest-5.x` The latest tagged version of Arizona Bootstrap 5
```

File: README.md [hunk 3]
```diff
@@ -25,4 +25,66 @@ Arizona Bootstrap is built off of Bootstrap 4, and UA Bootstrap is built off of
...
+- A pull request is merged into the `2.x` branch of this repository
+- A new release is created on the `main` or `2.x` branch.
+
+The URL structure for the CDN is as follows:
+```
+https://cdn.digital.arizona.edu/lib/arizona-bootstrap/<version bucket>/<asset type>/<filename>
+```
+
+#### The `<version bucket>` parameter can be any of the following
+- `main` Deprecated
```
Comment (trackleft) [inline README.md:48]:
```suggestion
- `main` Deprecated as of `2.0.20`
```

File: README.md [hunk 4]
```diff
@@ -25,4 +25,66 @@ Arizona Bootstrap is built off of Bootstrap 4, and UA Bootstrap is built off of
...
+- `2.x` Development version of Arizona Bootstrap 2
+- `5.x` Development version of Arizona Bootstrap 5
+- A release version number, Example: `2.0.19` See [the release
+page](https://github.com/az-digital/arizona-bootstrap/releases) for a full list
+of possible options. Remove the `v` from the version number.
+- `latest-2.x` The latest tagged version of Arizona Bootstrap 2
+- `latest-5.x` The latest tagged version of Arizona Bootstrap 5
+- `latest` Deprecated
```
Comment (trackleft) [inline README.md:56]:
```suggestion
- `latest` Deprecated as of `2.0.20`

```

File: README.md [hunk 5]
```diff
@@ -25,4 +25,66 @@ Arizona Bootstrap is built off of Bootstrap 4, and UA Bootstrap is built off of
...
+
+**NOTE: `main` is deprecated and will no longer be updated as of `v2.0.20` - to
+use the latest development version of the `2.x` branch use `2.x`**
+
+**NOTE: `latest` is deprecated and will no longer be updated as of `v2.0.20` -
+to use the latest stable version of the `2.x` branch use `latest-2.x`**
```
Comment (trackleft) [inline README.md:62]:
```suggestion
to use the latest tagged version of the `2.x` branch use `latest-2.x`**
```

File: README.md [hunk 6]
```diff
@@ -25,4 +25,66 @@ Arizona Bootstrap is built off of Bootstrap 4, and UA Bootstrap is built off of
...
+- Update `latest-5.x` bucket
+- Create a new bucket for the version that was released. Example: `5.0.0`
+
+### Release is tagged on the `2.x` branch
+All tagged releases on the `2.x` branch will:
+- Update `latest-2.x` bucket
+- Create a new bucket for the version that was released. Example: `2.0.20`
+
+### Pull request is merged into `main` branch
+All code merged into the `main` branch will update `5.x` bucket.
```
Comment (bberndt-uaz) [inline README.md:75]:
```suggestion
All code merged into the `main` branch will update the `5.x` bucket.
```

File: README.md (hunk 3 above)
Comment (joeparsons) [reply README.md:48]:
```suggestion
- `main` Deprecated (no longer updated) as of `2.0.20`
```

File: README.md (hunk 4 above)
Comment (joeparsons) [reply README.md:56]: Nitpick:
```suggestion
- `latest` Deprecated (no longer updated) as of `2.0.20`
```

**Review by joeparsons — DISMISSED** [review]
This all looks good to me (including the commit suggestions)

**Review by joeparsons — APPROVED** [review]

**Review by bberndt-uaz — APPROVED** [review]

**Review by mmunro-ltrr — APPROVED** [review]

**Pull request merged**